src/telliot_feeds/reporters/tips/listener/utils.py

```diff
@@ -0,0 +1,62 @@
+from collections import defaultdict
+from typing import Any
+from typing import Dict
```

Comment:
Don't need some of these imports, can just use dict, list, and tuple for type hints